.github/workflows/gradual-strict-null-checks.yml

```
@@ -66,7 +66,7 @@ jobs:
           MAIN=$(grep "Found [0-9]* errors" .out-main | sed 's/Found \(.*\) errors in .* files./\1/')
 
           if [ $CURRENT -gt $MAIN ]; then
-            comment "After enabling strictNullChecks this PR would be increasing the number of null check errors from ${MAIN} to ${CURRENT}. Make sure your branch is up-to-date with ${MAIN_BRANCH} and check the diff in the console output to gather more details"
+            comment "After enabling \`strictNullChecks\` this PR would be **increasing** the number of null check errors from ${MAIN} to ${CURRENT}. <br /> Make sure your branch is up-to-date with ${MAIN_BRANCH} and **check the diff in the console output** to gather more details.<br /><br />**So what are strict null checks?** When strictNullChecks is \`false\`, null and undefined are effectively ignored by the language. This can lead to unexpected errors at runtime.<br/>When strictNullChecks is \`true\`, null and undefined have their own distinct types and you'll get a type error if you try to use them where a concrete value is expected.<br>[Documentation](https://www.typescriptlang.org/tsconfig#strictNullChecks)"
```

```suggestion
            comment "After enabling \`strictNullChecks\` this PR would be **increasing** the number of null check errors from ${MAIN} to ${CURRENT}. <br /> Make sure your branch is up-to-date with ${MAIN_BRANCH} and **check the diff in the console output** to gather more details.<br /><br />## What are strict null checks?<br/><br/>From the [`strictNullChecks` reference documentation](https://www.typescriptlang.org/tsconfig#strictNullChecks):<br/><br/>When \`strictNullChecks\` is \`false\`, \`null\` and \`undefined\` are effectively ignored by the language. This can lead to unexpected errors at runtime.<br/>When \`strictNullChecks\` is \`true\`, \`null\` and \`undefined\` have their own distinct types and you'll get a type error if you try to use them where a concrete value is expected."
```

---

How about something like this? Moving the link to the docs to the front and clarifying that that's where the content is from?

---

Sure! We just need to escape the `strictNullChecks` inside the reference documentation link

---

Or we could do a simpler version to keep the message short (less text to read):

```suggestion
            comment "After enabling [\`strictNullChecks\`](https://www.typescriptlang.org/tsconfig#strictNullChecks) this PR would be **increasing** the number of null check errors from ${MAIN} to ${CURRENT}. <br /> Make sure your branch is up-to-date with ${MAIN_BRANCH} and **check the diff in the console output** to gather more details."
```